lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -362,9 +389,7 @@ putSignedTx inputs outputs witnesses = do
   where
     -- Assumes the `TxWitness` has been faithfully constructed
     putWitness :: TxWitness -> Put
-    putWitness (TxWitness bytes) = do
-        putWord8 1
-        putByteString bytes
+    putWitness (TxWitness bytes) = putByteString bytes
```

Comment:
The tag is now part of the `bytes` themselves (cf `utxoWitness` and `legacyUtxoWitness`)